spring-config/context-sparql.xml

```diff
@@ -13,146 +13,146 @@
-        
-        <!--  
-	        <bean id="uriCollectorBean" class="org.aksw.simba.squirrel.collect.SqlBasedUriCollector" >
-	        	<constructor-arg index="0" ref="datasourceConnection" />
-	        	<constructor-arg index="1" ref="serializerBean" />
-	        </bean>
-        -->
-        <bean id="serializerBean" class="org.dice_research.squirrel.data.uri.serialize.java.GzipJavaUriSerializer"/>
-        
-        <bean id="datasourceConnection" class="org.springframework.jdbc.datasource.DriverManagerDataSource" >
-        	<property name="driverClassName" value="org.hsqldb.jdbc.JDBCDriver" />
-        	<property name="url" value="jdbc:hsqldb:mem:foundUris" />
-        	<property name="username" value="SA" />
-        	<property name="password" value="" />
-        </bean>
-        
-        <bean id="robotsManagerBean" class="org.dice_research.squirrel.robots.RobotsManagerImpl">
-        	<constructor-arg index="0" ref="simpleHttpFetcherBean" />
-        	<property name="defaultMinWaitingTime" value="100"></property>
-        </bean>
-    
-    <bean id="simpleHttpFetcherBean" class="crawlercommons.fetcher.http.SimpleHttpFetcher" >
-        	<constructor-arg index="0" ref="userAgentBean" />
-        </bean>
-        
-        <bean id="userAgentBean" class="crawlercommons.fetcher.http.UserAgent" >
-        	<constructor-arg index="0" value="Test" />
-        	<constructor-arg index="1" value="" />
-        	<constructor-arg index="2" value="" />
+	</bean>
+
+
+	<bean id="uriCollectorBean" class="org.dice_research.squirrel.collect.SimpleUriCollector" >
+		<constructor-arg index="0" ref="serializerBean" />
+	</bean>
+
+	<!--
+        <bean id="uriCollectorBean" class="org.aksw.simba.squirrel.collect.SqlBasedUriCollector" >
+            <constructor-arg index="0" ref="datasourceConnection" />
+            <constructor-arg index="1" ref="serializerBean" />
         </bean>
-        
+    -->
+	<bean id="serializerBean" class="org.dice_research.squirrel.data.uri.serialize.java.GzipJavaUriSerializer"/>
+
+	<bean id="datasourceConnection" class="org.springframework.jdbc.datasource.DriverManagerDataSource" >
+		<property name="driverClassName" value="org.hsqldb.jdbc.JDBCDriver" />
+		<property name="url" value="jdbc:hsqldb:mem:foundUris" />
+		<property name="username" value="SA" />
+		<property name="password" value="" />
+	</bean>
+
+	<bean id="robotsManagerBean" class="org.dice_research.squirrel.robots.RobotsManagerImpl">
+		<constructor-arg index="0" ref="simpleHttpFetcherBean" />
+		<property name="defaultMinWaitingTime" value="100"></property>
+	</bean>
+
+	<bean id="simpleHttpFetcherBean" class="crawlercommons.fetcher.http.SimpleHttpFetcher" >
+		<constructor-arg index="0" ref="userAgentBean" />
+	</bean>
+
+	<bean id="userAgentBean" class="crawlercommons.fetcher.http.UserAgent" >
+		<constructor-arg index="0" value="Test" />
+		<constructor-arg index="1" value="" />
+		<constructor-arg index="2" value="" />
+	</bean>
+
 </beans>
```

Comment:
It looks like you haven't changed the content of the file but the formatting. Changes like that shouldn't be commited at all. Please clarify whether you have changed the files content.

Reply:
Aakash had made changes to use FileBasedSink for his regular crawling task but I have reverted back the file content similar to that of the dice-group Squirrel.

Reply:
https://github.com/abhihc/Squirrel/commit/46f55dffa0a0be0c68191e33daa22e4a34457627

Commit ID

Reply:
The problem is that its formatting is different compared to the "original file". If you want to revert changes, it might be better to check out the "original file" and overwrite your current file with this original file. However, you should check whether this old file really contains all the definitions you need.

Reply:
OK so the issue has been resolved.

Reply:
Thanks! That makes it much easier to concentrate on the changes you made :wink: